corehq/apps/user_importer/importer.py

```diff
@@ -490,7 +502,7 @@ def create_or_update_commcare_users_and_groups(upload_domain, user_specs, upload
                     status_row['flag'] = 'created'
 
                 if phone_numbers:
-                    phone_numbers = [n for n in phone_numbers if n]  # remove empty items
+                    phone_numbers = clean_phone_numbers(phone_numbers, 'Invalid phone number detected')
```

Comment:
nit: do you need to pass the error message? `clean_phone_numbers` is only used here and I think it can only pass that message. Plus would be nice for message to include the faulty number otherwise user has no way to tell what number caused the error

Reply:
Good point